client/src/constants.js

```diff
@@ -0,0 +1,3 @@
+const distantFuture = 2147483647;
+
+export default distantFuture;
```

Comment:
probably don't export this as default cause we expect other constants to end up here. this is fine for now